.sdkmanrc

```diff
@@ -1,3 +1,3 @@
 # Enable auto-env through the sdkman_auto_env config
 # Add key=value pairs of SDKs to use below
-java=17.0.8-graalce
+java=21.0.4-graal
```

Comment:
This is not a CE version of the VM.

Reply:
It seems Java 21 is being used as of [recently](https://github.com/seqeralabs/tower-cli/blob/949e6ebbd3cc502d45306ad4f572695ecf1f98ac/build.gradle#L118-L121). In GitHub Actions, we are using GraalVM 21 "non-CE" [too](https://github.com/seqeralabs/tower-cli/blob/949e6ebbd3cc502d45306ad4f572695ecf1f98ac/.github/workflows/build.yml#L26-L30). Here, I've just updated `.sdkmanrc` to align them accordingly. In fact, with GraalVM 21 CE it doesn't seem to work.

![image](https://github.com/user-attachments/assets/b99f57e0-2a1c-443b-bb5e-567d189cfb14)

I'll start a discussion about this.

Reply:
We can use it under the new Graalvm license 

> Subject to the conditions in the license, including the License for Early Adopter Versions, the GFTC is intended to permit use by any user including commercial and production use. Redistribution is permitted as long as it is not for a fee

https://www.graalvm.org/faq/